### DISCUSSION TITLE
plugin does not appear in JetBrains marketplace in CLion (only TabNine!)

### DISCUSSION THREAD

**Discussion by btc** [community/community#7241] [Copilot Conversations] [answered]
Surprisingly, only TabNine appears. Is this a known issue?

<img width="749" alt="image" src="https://user-images.githubusercontent.com/734339/140236661-ca53c436-e054-4ba7-b597-a4f34112f48f.png">

**Comment by davecheney** [comment, marked as answer]
Which version of clion are you using?

**Reply by btc** [reply]
> Which version of clion are you using?

CLion 2021.1.2
Build #CL-211.7442.42, built on June 1, 2021
Subscription is active until February 22, 2022.
For educational use only.
Runtime version: 11.0.11+9-b1341.57 x86_64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
macOS 12.0.1
GC: G1 Young Generation, G1 Old Generation
Memory: 33000M
Cores: 16
Registry: ide.windowSystem.autoShowProcessPopup=true, run.processes.with.pty=TRUE
Non-Bundled Plugins: GrepConsole (11.10.211.6086.0), IdeaVIM (0.69), Key Promoter X (2021.1.1), com.4lex4.intellij.solarized (2.3.0), com.compilerexplorer.compilerexplorer (1.18), com.paperetto.dash (3.3), mobi.hsz.idea.gitignore (4.1.0), org.toml.lang (0.2.152.4026-211), some.awesome (1.14), org.rust.lang (0.4.152.4026-211)

**Reply by btc** [reply]
Updating to 2021.2.3 now...

**Reply by btc** [reply]
there it is now!

<img width="358" alt="image" src="https://user-images.githubusercontent.com/734339/140241007-151e80d4-d7db-4772-b735-3e39ae401f36.png">

**Reply by btc** [reply]
`2021.1.2` is not recent enough. 

`2021.2.3` satisfies the requirements.
